src/features/user/Widget/DonationLink/DonationLinkForm.tsx

```diff
@@ -0,0 +1,280 @@
+                  `${(option as LanguageType).languageName}`
+                }
+                isOptionEqualToValue={(option, value) =>
+                  (option as LanguageType).langCode ===
+                  (value as LanguageType).langCode
+                }
+                value={Languages}
+                renderInput={(params) => (
+                  <TextField
+                    {...params}
+                    name="languagedropdown"
+                    label="Language"
+                    placeholder={t('donationLink:Languages')}
+                  />
+                )}
+                renderOption={(props, option) => (
+                  <StyledAutoCompleteOption
+                    {...props}
+                    key={(option as LanguageType).langCode}
+                  >
+                    {(option as LanguageType).langCode !== 'auto' && (
+                      <span>{`${(option as LanguageType).langCode}`}</span>
+                    )}
+                    {` ${(option as LanguageType).languageName}`}
+                  </StyledAutoCompleteOption>
+                )}
+                onChange={(event, newLan) => {
+                  if (newLan) {
+                    setLanguage(newLan as LanguageType);
+                  }
+                }}
+              />
+            </InlineFormDisplayGroup>
+          </div>
+          <div className={styles.formSection}>
+            <div className={styles.formHeader}>
+              {t('donationLink:projectTitle')}
+            </div>
+            <ProjectSelectAutocomplete
+              handleProjectChange={handleProjectChange}
+              project={localProject}
+              projectList={projectsList || []}
+              active={true}
+            />
+          </div>
+          <div className={styles.formHeader}>
+            {t('donationLink:treeCounterTitle')}
+          </div>
+          <InlineFormDisplayGroup>
+            <h6>{t('donationLink:treeCounterSubtitle')}</h6>
+            <Switch
+              id="treeCounter"
+              name="treeCounter"
+              checked={isSupport}
+              onChange={() => {
+                setIsSupport(!isSupport);
+              }}
+              disabled={user.isPrivate}
+            />
+          </InlineFormDisplayGroup>
+          <InlineFormDisplayGroup>
+            <div className={styles.formHeader}>
+              {t('donationLink:testingTitle')}
+            </div>
+            <Switch
+              id="testing"
+              name="testing"
+              checked={isTesting}
+              onChange={() => {
+                setIsTesting(!isTesting);
+              }}
+              disabled={false}
+            />
+          </InlineFormDisplayGroup>
+          {isTesting && (
+            <>
+              <h6> {t('donationLink:testingModeSubtitle1')}</h6>
+              <h6>
+                {t('donationLink:testingModeSubtitle2')}
+                <a
```

Comment:
Keep this link in the primary color. I think there's already a class `planet-links` defined in `global.scss` that does this.